test/e2e_runner_test.go

```diff
@@ -0,0 +1,41 @@
+package test
+
+import (
+	"github.com/strimzi/strimzi-canary/test/service_manager"
+	"log"
+	"os"
+	"path"
+	"runtime"
+	"testing"
+)
+
+var (
+	serviceManager *service_manager.ServiceManager
+)
+
+// returning into the root from /test
+func init() {
+	_, filename, _, _ := runtime.Caller(0)
+	dir := path.Join(path.Dir(filename), "..")
+	err := os.Chdir(dir)
```

Comment:
can you explain why this code is needed as init?

Reply:
Move one step upper (to root of project) in path from /test. But I can simply count with that.